.changeset/six-suns-live.md

```diff
@@ -0,0 +1,11 @@
+---
+"@comet/blocks-admin": minor
+---
+
+Add `resolveDependencyPath()` to `BlockMethods` interface
+
+Blocks must now offer a `resolveDependencyPath()` method that returns a URL path based on the block's `state` and `jsonPath`. 
```

Comment:
```suggestion
Blocks can now offer a `resolveDependencyPath()` method that returns a URL path based on the block's `state` and `jsonPath`. 
```

It's not required, right?

Reply:
Every block has it. But you normally don't have to implement it manually since it's in the block skeleton.

Reply:
It's required in the BlockMethods interface. I hope that's not breaking? 
https://github.com/vivid-planet/comet/blob/54abb712cc1fc97418c8f3d61335a4074982103d/packages/admin/blocks-admin/src/blocks/types.tsx#L87